.snippets/code/builders/build/eth-api/dev-env/foundry/ERC20.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.8.0;
+
+// Import OpenZeppelin Contract
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+// This ERC-20 contract mints the specified amount of tokens to the contract creator
+contract MyToken is ERC20 {
+  constructor(uint256 initialSupply) ERC20("MyToken", "MYTOK") {
```

Comment:
indents should be 4 spaces for solidity files

Reply:
looks like this file was missed @themacexpert

Reply:
It's 4 spaces, I just didn't resolve the convo. Lmk if it doesn't appear to be 4 spaces on your side but that's what I'm seeing here.